packages/lit-dev-api/package.json

```diff
@@ -6,8 +6,20 @@
   "author": "Google LLC",
   "license": "BSD-3-Clause",
   "scripts": {
-    "build": "node ../lit-dev-tools-cjs/lib/api-docs/generate.js",
-    "build:watch": "chokidar 'lit/packages/*/src/**/*.ts' '../lit-dev-tools-cjs/lib/api-docs/**/*.js' -c 'npm run build' --initial"
+    "build": "wireit",
+    "build:watch": "npm run build -- watch"
```

Comment:
Eventually I would just drop the `:watch` commands, because `npm run build watch` just works for free, but makes sense to keep them in for the time being since people might still be using them.